### PR TITLE
Remove usage of logr and use directly zap as in all the other components

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Review by james-bebbington — APPROVED** [review]

File: receiver/awsxrayreceiver/go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxra
 go 1.14
 
 require (
-	github.com/aws/aws-sdk-go v1.35.17
+	github.com/aws/aws-sdk-go v1.35.20
```
Comment (jpkrohling) [inline receiver/awsxrayreceiver/go.mod:6]: Same here.

File: exporter/awsxrayexporter/go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxra
 go 1.14
 
 require (
-	github.com/aws/aws-sdk-go v1.35.17
+	github.com/aws/aws-sdk-go v1.35.20
```
Comment (jpkrohling) [inline exporter/awsxrayexporter/go.mod:6]: Doesn't it belong to another PR?

**Review by jpkrohling — COMMENTED** [review]
LGTM, with two comments:

- The title could be something like "Replace logr with zap in tracegen"
- There are two unrelated changes to go.mod. I'm OK in having them here, but perhaps you didn't mean to introduce them in this PR?

File: exporter/awsxrayexporter/go.mod (hunk 2 above)
Comment (bogdandrutu) [reply exporter/awsxrayexporter/go.mod:6]: `make gotidy` created them :)). Let me rebase.

Comment (bogdandrutu) [reply exporter/awsxrayexporter/go.mod]: All unexpected go.mod changes are gone.

File: receiver/awsxrayreceiver/go.mod (hunk 1 above)
Comment (bogdandrutu) [reply receiver/awsxrayreceiver/go.mod]: All unexpected go.mod changes are gone.

**Review by jpkrohling — APPROVED** [review]
